examples/osp-flatcar-aws.yaml

```diff
@@ -189,6 +189,22 @@ spec:
       EOF
       {{- end }}
 
+    configureProxyScript: |-
+      {{- if .HTTPProxy }}
```

Comment:
For flatcar you also need to include `update-engine.service` as a unit. I actually overlooked this here: 
https://github.com/kubermatic/operating-system-manager/blob/master/examples/osp-flatcar-aws.yaml#L382-L383 I assume this would fail since the service doesn't exist, or am I overlooking something? Please check this out https://github.com/kubermatic/machine-controller/blob/master/pkg/userdata/flatcar/provider.go#L508-L515

Reply:
You are right, that is the only thing pending for flatcar/proxy at the moment. Let me add that as well.